lib/pengine/Makefile.am

```diff
@@ -16,15 +16,15 @@ lib_LTLIBRARIES		= libpe_rules.la libpe_status.la
 ## SOURCES
 noinst_HEADERS		= variant.h pe_status_private.h
 
-libpe_rules_la_LDFLAGS	= -version-info 27:0:1
+libpe_rules_la_LDFLAGS	= -version-info 27:1:1
 
 libpe_rules_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
 libpe_rules_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
 libpe_rules_la_LIBADD	= $(top_builddir)/lib/common/libcrmcommon.la
 libpe_rules_la_SOURCES	= rules.c rules_alerts.c common.c
 
-libpe_status_la_LDFLAGS	= -version-info 29:0:1
+libpe_status_la_LDFLAGS	= -version-info 29:1:1
```

Comment:
79ded22 added new public struct members, so it should be 30:0:2

Reply:
of course ... was actually missing that ... must have forgotten to scroll down on the header-diff
added --no-pager to bumplibs

Reply:
It's a pain, I usually look back over the changes 3 times :) but after doing it a few releases, it starts to get easier. The colorization will help a lot :)

It's also a pain that libpe_status and libpe_rules share some code, makes it really difficult to keep straight. My general plan for the libraries is first to update all internal APIs to follow current naming/return code guidelines, then for public APIs, I'm thinking of adding new APIs following current guidelines (and making all pacemaker code use those) and making the old APIs wrappers. At that point we can also consolidate the libraries so that all the new APIs are in libcrmcommon and libpacemaker, and all the other libraries will just have old API wrappers kept around for external code. The specifics will likely change but that general idea is where I'd like to go. All that meaning to say, this should become easier at that point :)

Reply:
We need some visible separation of the library sections in addition (separation line in a still unused color) or as an alternative feed each library section into a pager. For the latter I'm not sure if it is worth the effort or even desirable as I had to look back at other libraries.
Of course I would have liked to look in the other direction at still to come libraries to understand the code some times as well. So it maybe even makes sense to create the report first and ask for the bumps afterwards. Or memorize the answer and allow to go back and forth. Or export the full report into a document that can afterwards be worked with in an editor with the possibility to add comments - with a formalized part so that it can be automatically parsed afterwards to create the bumps automatically. We could even put the commented report under version-control so that we can easier remember later on why we decided for one or the other.

Anyway what you got was the result of my first pass - after I had the colors, commits & PRs - where I had the impression the the parts I was most unsure were from Chris which made me share it.